### PR TITLE
Java examples

### PR DESCRIPTION
**Pull request by anushreeSonak**
Hi @amitsahu8 
Please Find attached Programs of Linedlist, MultipleArray, Multiple Interface, Treeset. ArrayList

Thanks!!

### CODE REVIEW

File: Country.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package firstPackage;
+
+import java.util.*;
+public class Country 
+{
+	public static void main(String[] args) 
+	{
+		//Initialize ArrayList
```
Comment (amitsahu8) [inline Country.java]: code format issue

File: Country.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		{
+			startCities.add(path.get(0));
+			endCities.add(path.get(1));
+		}
+		for(String endCity : endCities)
+		{
+			if (!startCities.contains(endCity))
+			{
+				return endCity;
+
```
Comment (amitsahu8) [inline Country.java]: remove extra spaces

File: Country.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+		
+		//Print destination City
+		System.out.println(destination);
+	}
+
+	
+	public static String findDestination(List<List<String>> paths) 
+	{
+		//Declare set variable
+		Set<String> startCities = new HashSet<>();
```
Comment (amitsahu8) [inline Country.java]: Why set and not any other like ArrayList

File: Example2.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	 public static void main(String[] args) {
+	        String s = "aaabb";
+	        boolean result = checkString(s);
+	        System.out.println(result);
+	    }
+
+	    public static boolean checkString(String s) {
+	        int[] count = new int[26];
+
+	        for (int i = 0; i < s.length(); i++) {
```
Comment (amitsahu8) [inline Example2.java]: Re-write the logic

File: First/src/tutorials/MultipleArray.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+public class MultipleArray {
+
+	public static void main(String[] args) 
+	{
+		int a [][]= new int[2][2]; //{{0,9},{5,1,6},{2,2,7}};
+		a[0][0]=1;
+		a[0][1]=2;
+		a[1][0]=3;
+		a[1][1]=4;
+
```
Comment (amitsahu8) [inline First/src/tutorials/MultipleArray.java]: Create a new method and pass the array as argument and use it in main function

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: Country.java (hunk 3 above)
Comment (anushreeSonak) [reply Country.java]: Applied ArrayList

File: Country.java (hunk 2 above)
Comment (anushreeSonak) [reply Country.java]: Removed Extra spaces.

File: Country.java (hunk 1 above)
Comment (anushreeSonak) [reply Country.java]: Formatting Done !!

File: First/src/tutorials/MultipleArray.java (hunk 5 above)
Comment (anushreeSonak) [reply First/src/tutorials/MultipleArray.java]: Done !

File: Example2.java (hunk 4 above)
Comment (anushreeSonak) [reply Example2.java]: Done ! I have added new file named as  EqualOccurance.java

**Comment by anushreeSonak** [comment]
Its not in use.

**Pull request closed without merging**